pages/user-created-bounty/[id]/index.tsx

```
@@ -0,0 +1,105 @@
+import { useDispatch } from 'react-redux';
+import { setNetwork } from '~src/redux/network';
+import LoadingState from '~src/basic-components/Loading/LoadingState';
+import { getUserCreatedBountyById } from 'pages/api/v1/user-created-bounties/getUserCreatedBounty';
+import UserBountyPage from '~src/components/UserCreatedBounties/UserBountyPage';
+import { IUserCreatedBounty } from '~src/types';
+
+export const getServerSideProps: GetServerSideProps = async ({ req, query }) => {
+	const { id } = query;
+
+	const bountyId = Number(id);
+	if (isNaN(bountyId) || bountyId <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)